Model/Order/Creditmemo/ItemPlugin.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * @by SwiftOtter, Inc. 4/19/17
+ * @website https://swiftotter.com
+ **/
+
+namespace SwiftOtter\ShippingSurcharge\Model\Order\Creditmemo;
+
+use SwiftOtter\ShippingSurcharge\Model\Surcharge as SurchargeModel;
+
+class ItemPlugin
+{
+    public function afterCalcRowTotal(\Magento\Sales\Model\Order\Creditmemo\Item $item)
```

Comment:
If a namespace is used only once in a class, is there a benefit to aliasing? My thought has been that in many cases it can ambiguity when reading the code. Also, since this is a plugin, I have preferred to put the FQDN directly in the parameter to make it clear. Thoughts?